firmware/controllers/math/pid.cpp

```diff
@@ -39,7 +39,7 @@ float Pid::getValue(float target, float input) {
 }
 
 float Pid::getRawValue(float target, float input, float dTime) {
-	float error = target - input;
+	float error = (target - input) * errorAmplificationCoef;
```

Comment:
This is needed for Idle-IAC PID regulator.
When rpm < targetRpm, there's a risk of dropping RPM too low - and the engine dies out.
So PID reaction should be somewhat increased by adding extra percent to PID-error (a guessed value is ~4% more).
This is a way to 'amplify' the error by a given coefficient. Normally it's 1.0, so nothing happens. 
But for low RPMs we set this coef to the user-defined value (ex. 1.04) - then PID thinks that RPM is lower than it is, and controls IAC more aggressively.

Reply:
This can be potentially useful for other PID applications as well - like in ETB (still needs testing), when some external force (ex. a spring) helps to move in one side and resists in the opposite direction.